tools/gce/linux_worker_init.sh

```diff
@@ -34,6 +34,14 @@
 
 set -ex
 
+# Create some swap space
+sudo dd if=/dev/zero of=/swap bs=1024 count=10485760
```

Comment:
The creation script says "1TB", and this is basically the first command we run :)